### PR TITLE
ctl, master: support multiple sources create/stop

### LINKED ISSUE #516: HA: multiple sources create support in one command
## Feature Request

**Is your feature request related to a problem? Please describe:**

`operate-source create` can only create one source at a time now.

**Describe the feature you'd like:**

create multiple sources in one command.

**Describe alternatives you've considered:**

accept a directory path argument contain multiple source config files in one command.

**Teachability, Documentation, Adoption, Migration Strategy:**

### PR DESCRIPTION
**Pull request by lance6716**
### What problem does this PR solve? 
fix #516

### What is changed and how it works?
support multiple sources create/stop in one command

### Check List 

Tests 

 - modify unit test to test this command

Code changes

Side effects

Related changes

 - Need to update the documentation
 - Need to be included in the release note

### CODE REVIEW

**Comment by lance6716** [comment]
/run-all-tests

tests seem very unstable, in two successive CI (triggered by /run-all-tests), 
former has these failed unit tests:
FAIL: server_test.go:1289: testMaster.TestOfflineMember
FAIL: server_test.go:277: testMaster.TestQueryStatus
FAIL: server_test.go:379: testMaster.TestStartTask
FAIL: server_test.go:455: testMaster.TestStartTaskWithRemoveMeta
FAIL: server_test.go:886: testMaster.TestSwitchWorkerRelayMaster
FAIL: optimist_test.go:147: testOptimist.TestOptimist
latter has these plus integration test ha_cases:
FAIL: operation_test.go:38: testForEtcd.TestOperationEtcd

**Comment by lance6716** [comment]
/run-all-tests

those failed tests can pass in my local PC 😭

**Comment by WangXiangUSTC** [comment]
the `ha_cases` run failed sometimes on my local computer, it seems that should wait more time here https://github.com/pingcap/dm/blob/7a0038d5bb8e97398d1eb64ad88491d48ffac92a/tests/ha_cases/run.sh#L73, or check_sync_diff try more than 10 times, it should help reduce the probability of error

File: dm/ctl/master/operate_source.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ import (
 // NewOperateSourceCmd creates a OperateSource command
 func NewOperateSourceCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "operate-source <operate-type> <config-file>",
+		Use:   "operate-source <operate-type> <config-file> [config-file ...]",
 		Short: "create/update/stop upstream MySQL/MariaDB source",
```
Comment (GMHDBJD) [inline dm/ctl/master/operate_source.go:30]: Maybe `operate-source <operate-type> <config-file> [config-file ...]`

File: dm/ctl/master/operate_source.go [hunk 2]
```diff
@@ -48,31 +48,36 @@ func convertCmdType(t string) pb.SourceOp {
...
 		common.PrintLines("invalid operate '%s' on worker", cmdType)
 		return
 	}
 
+	contents := make([]string, len(cmd.Flags().Args())-1)
+	for i := 1; i < len(cmd.Flags().Args()); i++ {
+		configFile := cmd.Flags().Arg(i)
+		content, err := common.GetFileContent(configFile)
+		if err != nil {
+			common.PrintLines("get file content error:\n%v", err)
```
Comment (GMHDBJD) [inline dm/ctl/master/operate_source.go:69]: Should we also print filename which is failed?

File: dm/ctl/master/operate_source.go (hunk 1 above)
Comment (lance6716) [reply dm/ctl/master/operate_source.go:30]: want to express "at lease one `config-file`"

File: dm/ctl/master/operate_source.go (hunk 2 above)
Comment (lance6716) [reply dm/ctl/master/operate_source.go:69]: original `err` should already contain filename

File: go.mod [hunk 3]
```diff
@@ -56,7 +56,7 @@ require (
 	golang.org/x/net v0.0.0-20200421231249-e086a090c8fd // indirect
 	golang.org/x/sys v0.0.0-20200420163511-1957bb5e6d1f
 	golang.org/x/time v0.0.0-20191024005414-555d28b269f0
-	golang.org/x/tools v0.0.0-20200422205258-72e4a01eba43 // indirect
+	golang.org/x/tools v0.0.0-20200702044944-0cc1aa72b347 // indirect
```
Comment (GMHDBJD) [inline go.mod:59]: Is this necessary

Comment (lance6716) [reply go.mod:59]: forget what command changes it, maybe GoLand sync dependency, try to revert 😂

File: dm/master/server.go [hunk 4]
```diff
@@ -1327,64 +1332,112 @@ func (s *Server) OperateSource(ctx context.Context, req *pb.OperateSourceRequest
...
+			}
+			started = append(started, cfg.SourceID)
+		}
+
+		if hasError {
+			log.L().Info("reverting start source", zap.String("error", err.Error()))
+			for _, sid := range started {
+				err := s.scheduler.RemoveSourceCfg(sid)
+				if err != nil {
+					log.L().Error("while reverting start source, another error happens",
```
Comment (WangXiangUSTC) [inline dm/master/server.go:1367]:
```suggestion
					log.L().Error("while reverting started source, another error happens",
```

**Comment by GMHDBJD** [comment]
Please resolve conflict.

**Pull request merged**